arch/cortex-m/src/mpu.rs

```diff
@@ -275,7 +275,11 @@ impl CortexMRegion {
         region_num: usize,
         subregions: Option<(usize, usize)>,
         permissions: mpu::Permissions,
-    ) -> CortexMRegion {
+    ) -> Option<CortexMRegion> {
+        if logical_size < 8 || region_size == 0 {
```

Comment:
Sorry for the delay, I've been busy attending a conference the last week. I have added a name for the constant being used, and made the check a little tighter (logical size should never be less than 32 bytes, since that is the smallest possible MPU region/subregion for any CortexM core).